src/mame/layout/j_lal.lay

```diff
@@ -0,0 +1,409 @@
+<?xml version="1.0"?>
+<!--
+Lite a Line
+license:CC0
+-->
+<mamelayout version="2">
+	<element name="matrixlamp">
+		<rect state ="0">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="0.7" green="0.7" blue="0.7" />
+		</rect>
+		<rect state ="1">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="0.95" green="0.95" blue="0.43" />
+		</rect>
+	</element>
+	<element name="rect_white">
+		<rect>
+			<color red="1.0" green="1.0" blue="1.0"/>
+		</rect>
+	</element>
+	<element name="rect_black">
+		<rect>
+			<color red="0.0" green="0.0" blue="0.0"/>
+		</rect>
+	</element>
+	<element name="text_credits">
+		<text string="Credits">
+			<color red="1.0" green="1.0" blue="1.0"/>
+			<bounds x="0.05" y="0.05" width="0.90" height="0.90"/>
+		</text>
```

Comment:
Same for these text elements – if you want to inset the text from the bounds of the element, you need to add a transparent rectangle to force the bounds (e.g. <rect><color alpha="0" /></rect>`).  The way you have it now, the elements bounds will be (0.05 0.05 0.95 0.95) so there will be no space around the text anyway.

Reply:
Removed them, copypaste as well.